packages/react/components/modal/Modal.tsx

```
@@ -46,55 +46,24 @@ const Modal = ({
-      document.addEventListener('mousedown', handleClickOutside)
-    } else {
-      document.removeEventListener('mousedown', handleClickOutside)
-    }
-    return () => {
-      document.removeEventListener('mousedown', handleClickOutside)
-    }
-  }, [display, unClosable])
+  const handleClose = React.useCallback(
+    (onCloseFunc: ClickEvent | undefined, e: OnClickEvent) => {
+      setDisplay(false)
+      refBtnModal.current && refBtnModal.current.focus()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)